packages/connection-encrypter-tls/src/pb/index.proto

```diff
@@ -3,7 +3,7 @@ syntax = "proto3";
 enum KeyType {
   RSA = 0;
   Ed25519 = 1;
-  Secp256k1 = 2;
+  secp256k1 = 2;
```

Comment:
why?

Reply:
It's referred to as `secp256k1` in [it's paper](https://www.secg.org/sec2-v2.pdf), plus where we have it as [a string constant](https://github.com/libp2p/js-libp2p/blob/main/packages/interface/src/keys/index.ts#L37) we have it as "secp256k1" so it's just bringing this in line with that.

On the wire `2` is still `2` so there's no difference.

Reply:
should include it in the breaking changes list